### PR TITLE
refactor: clean organisms stories

### LINKED ISSUE #427: [Chore] Clean Organisms stories
Clean Organism Stories and add more example
Like here -> #379

### PR DESCRIPTION
**Pull request by pspaczek**
# Related issue

Closes #427
# Scope of work

- [x] SfAccordion
- [x] SfBannerGrid *need 100% refactor
- [x] SfBottomNavigation *after #706 
- [x] SfCarousel *need glide refactotor, slide by page
- [x] SfCollectedProduct
- [x] SfContentPages
- [x] SfFooter
- [x] SfGroupedProduct 
- [x] SfHeader
- [x] SfHero
- [x] SfList
- [x] SfMegaMenu
- [x] SfProductCard
- [x] SfSidebar
- [x] SfStoreLocator
- [x] SfTable
- [x] SfTabs
- [x] SfTopBar
# Screenshots of visual changes

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 1]
```diff
@@ -1,55 +1,38 @@
...
-  justify-content: space-around;
+  flex-direction: column;
+  justify-content: center;
   align-items: center;
-  height: 60px;
-  width: 100%;
-  padding: 0 1rem;
-  background: white;
-  box-shadow: 0px -2px 10px rgba(154, 154, 154, .15);
+  position: relative;
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]: `position` should go above` display`

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 2]
```diff
@@ -1,55 +1,38 @@
...
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    position: relative;
-    font-size: var(--bottom-navigation__item-font-size);
-    color: var(--c-gray);
+  & > .sf-icon {
+    --icon-color: var(--bottom-navigation-item-color);
+  }
+  &__lable{
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]:
```suggestion
  &__label {
```

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 3]
```diff
@@ -1,55 +1,38 @@
...
-      font-weight: var(--body-font-weight-secondary);
-
-      & > .sf-icon {
-        --icon-color: var(--bottom-navigation__item-color--active);
-      }
-    }
-
-    &--has-margin {
-      margin-top: var(--spacer);
+      --icon-color: var(--bottom-navigation__item-color--active);
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]:
```suggestion
      --icon-color: var(--bottom-navigation-item-color--active);
```

**Comment by aniamusial** [comment]
Screenshot from 2020-01-31 13-09-42
Something happened to tabs in SfContentPages

**Comment by aniamusial** [comment]
Screenshot from 2020-01-31 13-07-13
Active modifier is not applied, please add `:customClass="customClass`

**Comment by aniamusial** [comment]
Screenshot from 2020-01-31 13-06-24
Accordion styling collapsed

**Comment by aniamusial** [comment]
Screenshot from 2020-01-31 13-09-42

Input collapsed in GroupedProduct

**Comment by aniamusial** [comment]
Screenshot from 2020-01-31 13-22-28
Screenshot from 2020-01-31 13-22-17

SfHeader: 

- logo is huge again
- there is no padding on header

**Comment by aniamusial** [comment]
Error compiling template on SfHeader [slot] header-icons 
Screenshot from 2020-01-31 13-25-29

**Comment by pspaczek** [comment]
@aniamusial 
source

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 1 above)
Comment (pspaczek) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss]: +!

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 2 above)
Comment (pspaczek) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss]: +1

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss (hunk 3 above)
Comment (pspaczek) [reply packages/shared/styles/components/organisms/SfBottomNavigation.scss]: +1

**Pull request merged**